python-2.7.15/python-2.7.15.json

```diff
@@ -0,0 +1,46 @@
+{
+    "name": "python-2.7",
+    "sources": [
+        {
+            "type": "archive",
+            "url": "https://www.python.org/ftp/python/2.7.15/Python-2.7.15.tgz",
+            "sha256": "18617d1f15a380a919d517630a9cd85ce17ea602f9bbdc58ddc672df4b0239db"
+        }
+    ],
+    "buildsystem": "autotools",
+    "confgi-opts": [
+        "--enable-shared",
+        "--with-ensurepip=yes",
+        "--with-system-expat",
+        "--with-system-ffi",
+        "--enable-loadable-sqlite-extensions",
+        "--with-dbmliborder=gdbm",
+        "--enable-unicode=ucs4"
```

Comment:
Technically you could also pass `--enable-optimizations --enable-lto` but I've tested it only on x86_64. Does it use shared zlib too?

Reply:
I used the same config we use to build py2 in fd.o Sdk, we could try to enable those but haven't tested them.

> Does it use shared zlib too?
Yes, the zlib is in the .Platform of 18.08